.github/releases/wasm-v0.5.0-dev.6.md

```diff
@@ -0,0 +1 @@
+This release introduces a preview of stronghold bindings for the nodejs programming language. Stronghold offers best effort secure software storage of sensitive cryptographic key material written in Rust. To use the stronghold package install `@iota/identity-stronghold-nodejs` and follow the instructions of the package [README](https://github.com/iotaledger/identity.rs/tree/dev/bindings/stronghold-nodejs).
```

Comment:
We should add an example of using the Account with Stronghold in the Node.js bindings README.md, since it's linked above.

```suggestion
This release introduces preliminary Stronghold storage bindings for Node.js. Stronghold offers best-effort secure software storage for sensitive cryptographic key material written in Rust. To use the Stronghold storage package install `@iota/identity-stronghold-nodejs` and follow the instructions of the package [README](https://github.com/iotaledger/identity.rs/tree/dev/bindings/stronghold-nodejs). 

Note that all features related to diff chain updates have been marked as deprecated in this release. Diff chains are a useful optimisation when publishing many updates to a DID Document. However, their design may be incompatible with upcoming changes to the IOTA network and should be considered unstable.

Another major change is the removal of the `MerkleKeyCollection` verification method type which provided a compact representation for issuing and revoking Verifiable Credentials with multiple cryptographic keys. The `MerkleKeyCollection` suffered from disadvantages which limited scalability when managing more than a few thousand keys. While these disadvantages could be mitigated somewhat, the decision was made to replace it with one or more alternatives not affected by its fundamental limitations, upcoming in the next major release of 0.6.
```